src/peergos/shared/user/Snapshot.java

```diff
@@ -0,0 +1,56 @@
+package peergos.shared.user;
+
+import peergos.shared.*;
+import peergos.shared.crypto.*;
+import peergos.shared.crypto.hash.*;
+
+import java.util.*;
+import java.util.concurrent.*;
+
+public class Snapshot {
```

Comment:
Not convinced about the class name. Needs to be more descriptive

Reply:
I've added some javadoc